src/backend/opencl/blas_clblas.cpp

```diff
@@ -7,6 +7,8 @@
  * http://arrayfire.com/licenses/BSD-3-Clause
  ********************************************************/
 
+#if defined(USE_CLBLAS)
```

Comment:
This is not needed anymore since the `blas_*.cpp` files are merged into a single `blas.cpp` file. The only #ifdefs left over are in `magma_blas.h`.